runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/internal/validator/config/StringValidatorConfig.java

```diff
@@ -15,16 +15,37 @@
  */
 package io.aklivity.zilla.runtime.engine.internal.validator.config;
 
+import java.util.function.Function;
+
 import io.aklivity.zilla.runtime.engine.config.ValidatorConfig;
 
 public final class StringValidatorConfig extends ValidatorConfig
 {
+    public static final String DEFAULT_ENCODING = "utf_8";
+
     public final String encoding;
 
+    public StringValidatorConfig()
+    {
+        super("string");
+        this.encoding = DEFAULT_ENCODING;
+    }
+
```

Comment:
We don't need this constructor as we use the builder and it only requires the one with `encoding`, agree?

Reply:
fixed